pages/cockpit/installation-guide/installation-guide-introduction.adoc

```diff
@@ -12,3 +12,59 @@ The following sections explain how to prepare, install and configure your Cockpi
 
 NOTE: This Installation Guide provides information about how to install Gravitee.io components. For external components, such as
 MongoDB, see the relevant product documentation.
+
+Cockpit generates a certificate that will be used to authenticate installations (APIM/AM) that will connect. To create this certificate, you need to put private/public keys in keystore/truststore file and provide them using the following configuration attributes:
+
+[source,yaml]
+----
+keystore:
+  type: pkcs12
+  path: <path to the keystore file>
+  password: <password>
+truststore:
+  type: pkcs12
+  path: <path to the truststore file>
+  password: <password>
+----
+
+You also need a certificate to activate SSL for the WebSocket. Put this certificate in a `server.p12` Keystore and provide it through the following configuration attributes:
+
+[source,yaml]
+----
+ssl:
+  keystore:
+    type: pkcs12
+    path: <path to the keystore file>
+    password: <password>
+  truststore:
+    type: pkcs12
+    path: <path to the truststore file>
+    password: <password>
+----
+
+WARNING: Cockpit won't start if no keystore/truststore are provided.
+
+== Generate certificates
+
+WARNING: Certificate commands are intended to be used exclusively for local development purposes.
```

Comment:
If you don't have proper certificates yet, but still want to install Cockpit for dev purpose, you can generate some certificates locally. Please note that those certificates should not be used on production environments.

```suggestion
WARNING: The following commands are intended to be used exclusively for local development purposes.
```